.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+/.yarn-offline-mirror
```

Comment:
As discussed during hands-on, there is a reason why .yarn-offline-mirror files were not added to .gitignore.

Note: You’ll notice that your commit includes binaries in the .yarn-offline-mirror folder. That’s expected as the repository is configured to run Yarn offline for more reliable builds. Future tutorial steps that don’t install new packages won’t have .yarn-offline-mirror commit changes.